rotkehlchen/chain/ethereum/modules/ens/constants.py

```diff
@@ -1 +1,5 @@
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+
+
 CPT_ENS = 'ens'
+ENS_CPT_DETAILS = CounterpartyDetails(identifier=CPT_ENS, label='ens', image='ens.svg')
```

Comment:
This and the other stuff in the file

```suggestion
ENS_CPT_DETAILS: Final = CounterpartyDetails(identifier=CPT_ENS, label='ens', image='ens.svg')
```